rofs-filtered-invert.rc

```diff
@@ -7,7 +7,9 @@
 #
 # Please read the instructions in rofs-filtered.rc first since they apply here
 # as well.
-#
+
+type:LNK
```

Comment:
So what happens if there's a regular file called `type:LNK` in the file system? We need a robust way of telling the config file parser that this is not a regex.

Reply:
processed independently of and before regex so all configured regex still apply.

Reply:
If I have a regular file called `type:LNK`, and I want to show it, I would have to add a line in here with `type:LNK` on it. That file however will not be shown because the code treats `type:LNK` as a mode filter not as a filename filter.

Reply:
I opted to retain special file type syntax not looking like a regex and document how to handle files that mimic the special file type syntax with an explicit regex.